### ISSUE TITLE
Bandwidth api issue:Client Error

### ISSUE THREAD

**Issue by nikhil-231997** [floodlight/floodlight#804]
**Describe the bug**
Client Error: Not Found for url: http://localhost:8080/wm/statistics/bandwidth/00:00:00:00:00:00:00:02/0/json

**Expected behavior**Should get the statistics information

**Desktop (please complete the following information):**
 - Floodlight Version [ v1.2 ]
 - OS: [Ubuntu]

**Comment by xslizikm** [comment]
Same problem here, can't use statistics collector's REST API neither on master or v1.2. But for some reason it works on commit f499a51e29f870287382e28fbaa17e02cfd027d3

**Comment by xslizikm** [comment]
Yes I did. It's kinda weird, because last year around this time I used floodlight's statistcs collector REST API without any problem, but now it just didn't work on master branch. So I was looking for a commit where statistics collector REST API works and found it.
